.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,56 @@
+name: Run UnitTests
+on:
+  pull_request:
+    branches:
+      - dev
+    paths-ignore:
+      - 'ovos_classifiers/version.py'
+      - 'examples/**'
+      - '.github/**'
+      - '.gitignore'
+      - 'LICENSE'
+      - 'CHANGELOG.md'
+      - 'MANIFEST.in'
+      - 'readme.md'
+      - 'scripts/**'
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'ovos_classifiers/version.py'
+      - 'requirements/**'
+      - 'examples/**'
+      - '.gitignore'
+      - 'LICENSE'
+      - 'CHANGELOG.md'
+      - 'MANIFEST.in'
+      - 'readme.md'
+      - 'scripts/**'
+  workflow_dispatch:
+
+jobs:
+  unit_tests:
+    strategy:
+      max-parallel: 2
+      matrix:
+        python-version: [ 3.8, 3.9, '3.10', 3.11 ]
```

Comment:
Why the `'` around 3.10?

Reply:
otherwise, yaml parses it as a float and returns `3.1`